test/scripts/e2e_subs/e2e-app-simulate.sh

```diff
@@ -0,0 +1,93 @@
+#!/bin/bash
+
+date '+app-simple-test start %Y%m%d_%H%M%S'
+
+set -e
+set -x
+set -o pipefail
+set -o nounset
+export SHELLOPTS
+
+WALLET=$1
+
+# Directory of this bash program
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+
+gcmd="goal -w ${WALLET}"
+
+ACCOUNT=$(${gcmd} account list|awk '{ print $3 }')
+
+CONST_TRUE="true"
+CONST_FALSE="false"
+
+${gcmd} clerk send -a 10000 -f ${ACCOUNT} -t ${ACCOUNT} -o pay1.tx
+${gcmd} clerk send -a 10000 -f ${ACCOUNT} -t ${ACCOUNT} -o pay2.tx
+
+cat pay1.tx pay2.tx | ${gcmd} clerk group -i - -o grouped.tx
+
+# We first test transaction group simulation WITHOUT signatures
+RES=$(${gcmd} clerk simulate -t grouped.tx | jq '."would-succeed"')
+
+if [[ $RES != $CONST_FALSE ]]; then
```

Comment:
Can we also check the `missing-signature` field here  too?

Reply:
yes, adding a set of tests here

Reply:
see [missing signature check](https://github.com/algorand/go-algorand/pull/5213/commits/d10ed71057a26579777a7b8af28c26ed92d266b0)